### PR TITLE
fix: illegal pv warning

### PR DESCRIPTION
**Pull request by gahtan-syarif**
https://github.com/Disservin/fast-chess/issues/500

### CODE REVIEW

File: app/src/matchmaking/match/match.cpp [hunk 1]
```diff
@@ -323,8 +323,8 @@ bool Match::isUciMove(const std::string& move) noexcept {
 }
 
 void Match::verifyPvLines(const Player& us) {
-    const static auto verifyPv = [](Board board, const std::vector<std::string>& tokens, std::string_view info) {
-        const auto fen = board.getFen();
+    const static auto verifyPv = [this](Board board, const std::vector<std::string>& tokens, std::string_view info) {
```
Comment (Disservin) [inline app/src/matchmaking/match/match.cpp:326]: you cant capture this because it is static

Comment (Disservin) [reply app/src/matchmaking/match/match.cpp:326]: and even then this won't work because the uci_moves are not correct...

Comment (gahtan-syarif) [reply app/src/matchmaking/match/match.cpp:326]: yeah, im trying to find a way to get the fen and moves from the command builder

**Comment by Disservin** [comment]
https://github.com/Disservin/fast-chess/pull/502

**Pull request closed without merging**